Sources/WhisperKit/Core/Models.swift

```diff
@@ -218,6 +218,7 @@ public struct DecodingCache {
 }
 
 public enum ChunkingStrategy: String, CaseIterable {
+    case none
```

Comment:
is it necessary to add this case? can it be handled by being optional / non-optional?

Reply:
This is added so that the app can use it in a setting that doesn't have a nil default.